frontend/src/App.tsx

```diff
@@ -0,0 +1,106 @@
+/* eslint-disable react/jsx-no-undef */
+import { useQuery, gql, DocumentNode } from "@apollo/client";
+import React from "react";
+import { theme } from "@diamondlightsource/ui-components"
+import { ChakraProvider } from "@chakra-ui/react";
+import { LoadMoreTable } from "./components/LoadMoreTable";
+
+const GET_INFO: DocumentNode = gql`
+query pinInfo ($after: String) {
+  libraryPins(first: 2, after: $after) {
+    pageInfo {
+      hasPreviousPage,
+      hasNextPage,
+      startCursor,
+      endCursor
+    },
+    edges {
+      cursor
+      node {
+        barcode,
+        loopSize,
+        status
+      }
+    }
+  }
+}
+`;
+
+function DisplayPinInfo(): React.JSX.Element {
+  const { loading, error, data, fetchMore } = useQuery(GET_INFO);
+
+  if (loading) return <p>Loading...</p>;
+  if (error) return <p>Error : {error.message} {error.extraInfo}</p>;
+
+  // function to load more content and update query result
```

Comment:
These comments feel a little unjustified, some well named functions and variables should be able to achieve just as much